tests/bootstrap.php

```diff
@@ -1,18 +1,3 @@
 <?php
 
-$baseDir   = dirname(__DIR__);
-$vendorDir = $baseDir . '/vendor';
-
-$loader = require_once $vendorDir . '/autoload.php';
-$loader->addClassMap(
-    array(
-        'Bartlett\Tests\Reflect\Analyser\FooAnalyser'
-            => __DIR__ . '/Analyser/FooAnalyser.php',
-        'Bartlett\Tests\Reflect\Analyser\BarAnalyser'
-            => __DIR__ . '/Analyser/BarAnalyser.php',
-        'Bartlett\Tests\Reflect\Model\GenericModelTest'
-            => __DIR__ . '/Model/GenericModelTest.php',
-        'Bartlett\Tests\Reflect\Environment\YourLogger'
-            => __DIR__ . '/Environment/YourLogger.php',
-    )
-);
+require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
Using the `composer` autoloader can load the all classes including test classes automatically.